mkdocs.yml

```
@@ -8,6 +8,8 @@ theme:
 nav:
   - Home: index.md
   - Base:
+    - Processos, Metodologias e Abordagens:
+      - Metodologia: metodologia.md
```

Se possível colocar o doc de metodologia dentro da pasta base/
```suggestion
    - Processos, Metodologias e Abordagens:
      - Metodologia: base/metodologia.md
```

---

Opa, consertado !